Phase05/InvertedIndex/Enums/TokenTypeEnum.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace InvertedIndex.Enums;
+
+public enum TokenType
+{
+    And,
+    Or,
+    Not
+}
+
+public static class TokenMethods
+{
+
+    public static TokenType GetTokenType(this string token)
+    {
+        if (token.StartsWith("+"))
+        {
+            return TokenType.Or;
+        }
+        if (token.StartsWith("-"))
+        {
+            return TokenType.Not;
+        }
+
+        return TokenType.And;
+    }
```

Comment:
Violates open close principle
What if a fourth type of word is added that needs to be handled differently?